app/models/services/facebook.rb

```diff
@@ -23,10 +23,14 @@ def post_to_facebook(url, body)
   end
 
   def create_post_params(post)
-    message = post.message.plain_text_without_markdown
-    if post.photos.any?
-      message += " " + short_post_url(post, protocol: AppConfig.pod_uri.scheme,
-                                            host: AppConfig.pod_uri.authority)
+    postMessage = post.message.plain_text_without_markdown
+
+    if postMessage.size == 0 && !post.photos.any?
```

Comment:
- Use `empty?` instead of `size == 0`.
- Use `none?` instead of inverting `any?`.